### PR TITLE
Remove bower

### PR DESCRIPTION
**Pull request by abbotto**
- Remove Bower

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bluedrop-l20n-shell",
-  "version": "1.0.2",
+  "version": "1.0.0",
```
Comment (jurko-gospodnetic) [inline package.json:3]: Why the move backwards in versions?

File: bower.json [hunk 2]
```diff
@@ -1,17 +0,0 @@
...
-  "description": "Shell bindings for the L20n framework",
-  "main": "l20n.js",
-  "keywords": [
-    "localization",
-    "l10n",
-    "l20n"
-  ],
-  "license": "Apache 2.0",
-  "homepage": "http://l20n.org"
-}
```
Comment (jurko-gospodnetic) [inline bower.json:17]: How come the bower definition got removed? Just curious - did BLNP stop using bower to maintain its browser side packages?

File: package.json (hunk 1 above)
Comment (gayan415) [reply package.json:3]: Fair point

**Comment by abbotto** [comment]
Bower is going to be replaced with NPM in bln-web - this package is being
updated as part of the process.

**Pull request merged**